### DISCUSSION TITLE
Popups with an entry control & return of the entered data

### DISCUSSION THREAD

**Discussion by davefxy** [CommunityToolkit/Maui#1970] [General]
Any thoughts of Popups with an entry control & return of the entered data like Mopops

**Comment by bijington** [comment]
That's entirely possible with our Popups. You can use the ShowPopupAsync method that returns a value

**Comment by davefxy** [comment]
You guys are doing an outstanding job, Thank you
________________________________

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
